modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm

```diff
@@ -619,6 +619,11 @@ GLOBAL_LIST_INIT(loadout_exosuits, generate_loadout_items(/datum/loadout_item/su
 	item_path = /obj/item/clothing/suit/british_officer
 	restricted_roles = list(JOB_WARDEN, JOB_DETECTIVE, JOB_SECURITY_OFFICER, JOB_HEAD_OF_SECURITY, JOB_CORRECTIONS_OFFICER)
 
+/datum/loadout_item/suit/secjacket
```

Comment:
Modularize this. We have our own loadout folders.

Reply:
> Modularize this. We have our own loadout folders.

Oh, didn't realize I used Skyrat's.

Reply:
> Modularize this. We have our own loadout folders.

It's fixed.